blazor/scheduler/data-binding.md

```diff
@@ -721,6 +721,207 @@ The following sample code demonstrates notifying user when server-side exception
     }
 }
 ```
+## Load on demand
+
+In the Scheduler, there is an option to implement data loading on demand, which helps minimize the amount of data transmitted over the network, especially when working with large volumes of data.This can be achieved by filtering appointments on the server side based on  start date and end date.
```

Comment:
Remove the unwanted space.
![image](https://github.com/syncfusion-content/blazor-docs/assets/42572585/943c8eb6-4a61-4303-af35-e8e8322662fa)

Reply:
Addressed the review corrections